mshadow/random.h

```diff
@@ -152,24 +272,39 @@ class Random<cpu, DType> {
   std::mt19937 rnd_engine_;
   /*! \brief random number seed used in random engine */
   unsigned rseed_;
-  // implementing generators.
-  inline void GenUniform(DType *dptr, index_t size, DType a, DType b) {
```

Comment:
No longer referenced. All distributions use the unified generic generator SampleDistribution.